modules/platform/nuxeo-platform-usermanager/src/main/java/org/nuxeo/ecm/platform/usermanager/UserManagerImpl.java

```diff
@@ -685,7 +685,7 @@ protected NuxeoGroup makeGroup(DocumentModel groupEntry) {
     }
 
     protected void appendSubgroups(String groupId, Set<String> groups, DocumentModel context) {
-        List<String> groupsToAppend = getGroupsInGroup(groupId, context);
+        List<String> groupsToAppend = getDescendantGroups(groupId);
         groups.addAll(groupsToAppend);
         for (String subgroupId : groupsToAppend) {
             groups.add(subgroupId);
```

Comment:
Sorry I missed this in the first round of review but the rest of the method seems completely redundant now. Could you check?

Reply:
Indeed it doesn't do anything, I'm going to update the PRs